### PR TITLE
CB-3402 task info api

### PR DESCRIPTION
**Pull request by alexander-skoblikov**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java [hunk 1]
```diff
@@ -146,6 +146,7 @@ public class StreamTransferConsumer implements IDataTransferConsumer<StreamConsu
     private StatOutputStream statStream;
     
     public StreamTransferConsumer() {
+        System.out.println();
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java]:
```suggestion
```

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/serialize/SerializerDescriptor.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public class SerializerDescriptor extends AbstractDescriptor
 
         this.id = config.getAttribute("id");
         this.serializerType = new ObjectType(config.getAttribute("class"));
-        this.distributed = CommonUtils.getBoolean(config.getAttribute("distributed"), true);
+        this.distributed = CommonUtils.getBoolean(config.getAttribute("supportsDistributedMode"), true);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/serialize/SerializerDescriptor.java:44]: Please use `RegistryConstants#ATTR_SUPPORTS_DISTRIBUTED_MODE`

Comment (alexander-skoblikov) [reply plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/serialize/SerializerDescriptor.java:44]: there is no dependence on `org.jkiss.dbeaver.registry`

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/serialize/SerializerDescriptor.java:44]: Oh. Okay

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**